common/data/src/main/java/org/thingsboard/mqtt/broker/common/data/StringUtils.java

```diff
@@ -188,10 +188,16 @@ public static String randomAlphabetic(int count) {
     }
 
     public static String generateSafeToken(int length) {
-        byte[] bytes = new byte[length];
-        RANDOM.nextBytes(bytes);
+        byte[] bytes = generateSafeTokenBytes(length);
         Base64.Encoder encoder = Base64.getUrlEncoder().withoutPadding();
         return encoder.encodeToString(bytes);
     }
+
+    public static byte[] generateSafeTokenBytes(int length) {
```

Comment:
Pls add this to BytesUtil with TODO to move BytesUtil class near the StringUtils